src/controller/base-stream-controller.js

```diff
@@ -125,4 +125,9 @@ export default class BaseStreamController extends TaskLoop {
     this.state = State.STOPPED;
     this.fragmentTracker = null;
   }
+
+  computeLivePosition (sliding, levelDetails) {
+    let targetLatency = this.config.liveSyncDuration !== undefined ? this.config.liveSyncDuration : this.config.liveSyncDurationCount * levelDetails.targetduration;
+    return sliding + Math.max(0, levelDetails.totalduration - targetLatency);
```

Comment:
@gregrealeyes Is it really totalduration and not totalDuration?

Reply:
I've confirmed that it is `totalduration` throughout the code base